relbench/tasks/stackex.py

```diff
@@ -361,7 +361,7 @@ class UsersInteractTask(RelBenchLinkTask):
     timedelta = pd.Timedelta(days=365 // 4)
 
     metrics = [link_prediction_precision, link_prediction_recall, link_prediction_map]
-    eval_k = 10
+    eval_k = 100
```

Comment:
How does this affect our MAP metric?

Reply:
The difference in mAP is:

with `eval_k=100`

- ID-GNN `0.03751`
- global-popularity `0.12440`

with `eval_k=10`

- ID-GNN `0.01780`
- global-popularity `0.11706`

Change was made simply to bring this task into line with the other two stackex link tasks, which also use `eval_k=100`.

Reply:
Makes sense! thanks